cmd/kops/toolbox_template.go

```diff
@@ -227,6 +233,20 @@ func newTemplateContext(files []string) (map[string]interface{}, error) {
 		}
 	}
 
+	// User specified a value via --set
+	for _, value := range values {
+		if err := strvals.ParseInto(value, context); err != nil {
+			return nil, fmt.Errorf("failed parsing --set data: %s", err)
```

Comment:
should this be `%v`?

Reply:
It shouldn't matter, according to golang's documentation on the [fmt package](https://golang.org/pkg/fmt/):
> If the format (which is implicitly %v for Println etc.) is valid for a string (%s %q %v %x %X), the following two rules apply:
> 4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).